<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generator_pass</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 255, 239);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="countPASS">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(237, 255, 248);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(212, 251, 255);</string>
    </property>
    <property name="text">
     <string>Количество паролей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(189, 229, 255);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Сгенерировать</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_int">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(212, 255, 254);</string>
    </property>
    <property name="text">
     <string>Цифры</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="count_symbol">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(237, 255, 248);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(212, 251, 255);</string>
    </property>
    <property name="text">
     <string>Количество символов</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_char">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(212, 255, 254);</string>
    </property>
    <property name="text">
     <string>Буквы</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_re">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(212, 255, 254);</string>
    </property>
    <property name="text">
     <string>Символы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(189, 229, 255);
font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
